recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue

```diff
@@ -14,6 +14,7 @@
     :button-width-size="buttonWidthSize"
     :text-class="textClass"
     :content-class="contentClass"
+    :open-popover="openPopover"
```

Comment:
This prop `:open-popover="openPopover"` is not working as expected.
Because if you toggle the `openPopover ` prop in the control section of storybook it is open it but not close.

The reason of this is this code probably:

```
showPopover () {
      if (!this.openPopover || this.open) {
        return false;
      }

      return this.arrowClick();
    },
```

I dont know if this is the desired behaviour.

Reply:
This is the expected behavior since it's not a toggle. Open is auto triggered on certain keywords. But close isn't, the user will manually close the popup.